### PR TITLE
MES-2824 | feat(ADI2) implement Journal slots for ADI2 tests

### PR DESCRIPTION
**Pull request by AndyPerryKainos**
## Description

- Implement Journal slot component variations for ADI2 tests.
- New AdditionalCandidateDetails component. 
- Fixed alignment of VehicleDetails component on iPad and in browser.
- Clean up of unused SCSS mixin and some notation standardisation.

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)

IMG_0013

IMG_0014

### CODE REVIEW

File: src/components/test-slot/additional-candidate-details/__tests__/additional-candidate-details.spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    component = fixture.componentInstance;
+  }));
+
+  describe('Class', () => {
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+  });
+
+  describe('DOM', () => {});
```
Comment (TomBillingtonUK) [inline src/components/test-slot/additional-candidate-details/__tests__/additional-candidate-details.spec.ts]: empty dom's can be deleted

File: src/components/test-slot/additional-candidate-details/additional-candidate-details.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  &__key {
+    display: inline-block;
+
+    &:nth-of-type(n+2) {
+      margin-left: 5px;
+
+      &::before {
+        content: '';
+        border-left: 1px solid black;
```
Comment (TomBillingtonUK) [inline src/components/test-slot/additional-candidate-details/additional-candidate-details.scss]: we should probably use gds-black here

**Review by TomBillingtonUK — APPROVED** [review]

File: src/components/test-slot/additional-candidate-details/additional-candidate-details.scss (hunk 2 above)
Comment (AndyPerryKainos) [reply src/components/test-slot/additional-candidate-details/additional-candidate-details.scss]: Checked elsewhere and couldn't see a variable in use for the surrounding text so used the exact same. I can switch it but will have to update others as well to ensure consistency.

File: src/components/test-slot/additional-candidate-details/__tests__/additional-candidate-details.spec.ts (hunk 1 above)
Comment (AndyPerryKainos) [reply src/components/test-slot/additional-candidate-details/__tests__/additional-candidate-details.spec.ts]: Will need deleting in some pre-existing components as well.

**Review by appleabs — APPROVED** [review]

**Pull request merged**